graph/db/graph_test.go

```diff
@@ -1054,6 +1105,64 @@ func newEdgePolicy(chanID uint64, updateTime int64) *models.ChannelEdgePolicy {
 	}
 }
 
+// TestAddEdgeProof tests the ability to add an edge proof to an existing edge.
+func TestAddEdgeProof(t *testing.T) {
+	t.Parallel()
+
+	graph, err := MakeTestGraph(t)
+	require.NoError(t, err, "unable to make test database")
```

Comment:
As a small refactor, we could make `MakeTestGraph(t)` either fail the test or simply return a graph instance, so we don’t need error checks in every test case. This could also be done in a follow-up.